packages/animations/example/lib/container_transition.dart

```diff
@@ -48,20 +48,86 @@ class OpenContainerTransformDemo extends StatefulWidget {
 
 class _OpenContainerTransformDemoState
     extends State<OpenContainerTransformDemo> {
+  ContainerTransitionType transitionType = ContainerTransitionType.fade;
+
   @override
   Widget build(BuildContext context) {
     return Scaffold(
-      appBar: AppBar(title: const Text('Container transform')),
+      appBar: AppBar(
+        title: const Text('Container transform'),
+        actions: <Widget>[
+          IconButton(
+            icon: Icon(Icons.settings),
+            onPressed: () {
+              showModalBottomSheet<void>(
+                context: context,
+                builder: (BuildContext context) {
+                  return StatefulBuilder(
+                    builder: (BuildContext context, StateSetter setModalState) {
+                      return Container(
+                        height: 125,
+                        padding: const EdgeInsets.all(15.0),
+                        child: Column(
+                          crossAxisAlignment: CrossAxisAlignment.start,
+                          children: <Widget>[
+                            Text(
+                              'Fade mode',
+                              style: Theme.of(context).textTheme.caption,
+                            ),
+                            const SizedBox(height: 12),
+                            ToggleButtons(
+                              borderRadius: BorderRadius.circular(2.0),
+                              selectedBorderColor:
+                                  Theme.of(context).colorScheme.primary,
+                              onPressed: (int index) {
+                                setState(() {
+                                  if (index == 0) {
```

Comment:
Maybe just write this as `transitionType = index == 0 ? ContainerTransitionType.fade : ContainerTransitionType.fadeThrough`?